src/pymovements/dataset/dataset.py

```diff
@@ -763,6 +770,25 @@ def compute_event_properties(
             if column != 'filepath'
         ]
 
+        if isinstance(event_properties, str):
+            event_property_names = [event_properties]
+        elif isinstance(event_properties, list):
+            event_property_names = event_properties
+        elif isinstance(event_properties, dict):
+            event_property_names = event_properties.keys()
+        elif isinstance(event_properties, tuple):
+            event_property_names = event_properties[1].keys()
```

Comment:
I just noticed, that it's probably much easier to reuse the properties of the initialized processor:

https://github.com/aeye-lab/pymovements/blob/e7d45528e8d8cbb1ead6eda447e158fc40947bde/src/pymovements/events/processing.py#L124

Here it is already guaranteed that it's a list of tuples, where the first item of each tuple is the name of the property and the second are optional keyword arguments.
Example: `[('dispersion', {}), ('location', {'method': 'median'})]`

So this way getting all the names is much easier:

```python
event_property_names = [property[0] for property in processor.event_properties]
```

Sorry that I previously lead you to this more complicated code.